packages/web/src/scss/components/FileUploader/README.md

```diff
@@ -189,17 +190,67 @@ truncated.
 </li>
 ```
 
+This way you can customize yourself as follows in `<template>`.
```

Comment:
I do not understand this sentence. What do you mean?

Reply:
"This way you can customize your own template in the `<template>` tag. " is better?

Reply:
Yes! Thanks, now I understand what you mean :)